data/remnant missions.txt

```diff
@@ -655,149 +655,288 @@ mission "Remnant: Learn Sign Follow Up"
+			goto end
+			label wave
+			`	The weapons technician finishes strapping down the plasma cannons and makes a gesture that you recognize as "farewell," then heads off toward the lab.`
+			label end
+			`	As you watch the remnant heading off with their new weapons to study, you can not help but wonder what will be the result of it.`
 
-			`	"I'll be careful," she sings with a tinge of sadness. "I am an engineer, not a Korath."`
-				goto more
 
-			label deep
-			`	"Fascinating," she sings as she clambers over the <ship> with the grace of someone who obviously knows her ships. "Above all else, I am an engineer of movement," she chants. "We have done well to hide as deep in the Ember Waste as we have, but our ships' limited ranges struggle to take us farther. Examining systems from elsewhere may inspire new developments."`
 
-			label more
-			`	After examining the <ship> for quite some time, she stops beneath it and looks up. "Is this a new form of ramscoop?" she asks as she gestures at your ship's ramscoop. As you nod in the affirmative, she continues. "Fuel has always been a great limiter of our exploration. If you could get a couple extra that we could dismantle, this could be very useful to us."`
+mission "Remnant: Catalytic Ramscoop"
+	name "Retrieve Catalytic Ramscoops for the Remnant"
+	description "A Remnant engineer has offered you <payment> for delivering two catalytic ramscoops to a lab on <planet>."
+	minor
+	source "Viminal"
+	destination
+		government "Remnant"
+	to offer
+		or
+			has "Remnant: Tech Retrieval: active"
+			has "Remnant: Tech Retrieval: done"
+	on offer
+		require "Catalytic Ramscoop"
+		conversation
+			`You recall that Taely was interested in seeing examples of new human technology, and the catalytic ramscoop definitely fits the criteria. Would you like to show her?`
 			choice
-				`	"Sure, I could bring you some."`
-				`	"Sorry, I'm really not interested in helping Remnant research."`
-					decline
-
-			`	After a short discussion, she agrees to pay you <payment> in exchange for two catalytic ramscoops.`
+				`	Not now.`
+					defer
+				`	Yes.`
+			branch stuff
+				has "Remnant Tech Retrieval: done"
+			`	After asking a few people, you finally get directed to a bay in the shipyard where you find Taely calibrating an inhibitor cannon on an albatross. "Greetings, <first>. I am just finishing these calibrations." She does a few more things inside the casing, reseals the hatch, and drops to the ground on a tether. "So, did you recover something?"`
+				goto the_rest
+			label stuff
+			`	Remembering that she spends most of her time in the shipyard, you head straight there and almost immediately are intercepted by another remnant who introduces herself as one of Taely's assistants. "Good day, Captain" he chants. "Our scans picked up a new form of ramscoop onboard your ship. Have you brought it for us to look at?"`
```

Comment:
This section:  "Remembering that she spends most of her time in the shipyard, you head straight there and almost immediately are intercepted by another remnant who introduces **herself** as one of Taely's assistants. "Good day, Captain" **he** chants." needs gender agreement (unless the character identifies as both genders which would be cool, but you'd need to explain that in backstory for most audiences.)

Reply:
That would be neat, but I probably won't get into that in my stories. I tend to ignore gender as a plot devise - at least as far as Endless Sky is concerned. 

Due to the fact that these missions have some variability to them (the player might be speaking to Taely or they might be speaking to one of her staff, depending on whether or not this is the first retrieval mission they do, or a subsequent one), all the staff in this set of missions are female. Otherwise too much text would have to be duplicated.